data/sql/updates/pending_db_world/rev_1620466647964636420.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620466647964636420');
+
+UPDATE `creature` SET `position_x`=-7133.56, `position_y`=-3161.51, `position_z`=244.252, `orientation` = 1.04837 WHERE `guid`=6947;
+UPDATE `creature` SET `position_x`=-6650.07, `position_y`=-3484.77, `position_z`=258.771, `orientation` = 1.16804 WHERE `guid`=7614;
+UPDATE `creature` SET `position_x`=-6649.06, `position_y`=-3482.49, `position_z`=259.517, `orientation` = 4.37896 WHERE `guid`=6915;
```

Comment:
```suggestion
UPDATE `creature` SET `position_x`=-6649.06, `position_y`=-3482.49, `position_z`=263.517, `orientation` = 4.37896 WHERE `guid`=6915;
```

6947 ✔️ perfect
7614 ✔️ at the server start it falls, after 2 seconds, it always gets the right position above the ground. 
6915 ❌ always fails underground, still needs a small adjustment of 4, 263 always works

@Hacki95 🐱